### ISSUE TITLE
manifest_version 3 extensions manifest.json with event pages should not be reported as invalid

### ISSUE THREAD

**Issue by rpl** [mozilla/addons-linter#4604]
On linting a manifest.json using a background event page:

```
{
	"manifest_version": 3,
	"name": "MV3 extension name",
	"version": "1.0.0",
	"background": {
		"page": "index.html"
	},
	"browser_specific_settings": {
		"gecko": {
			"id": "an-mv3-extension@test"
		}
	}
}
```

The addons-linter is currently reporting the manifest as invalid and mentioning that the background.service_worker property is required:
```
../addons-linter/bin/addons-linter --max-manifest-version=3 .
Validation Summary:

errors          1
notices         0
warnings        0

ERRORS:

Code                      Message                                     Description                                                   File            Line   Column
MANIFEST_FIELD_REQUIRED   "/background" must have required property   See https://mzl.la/1ZOhoEN (MDN Docs) for more information.   manifest.json
                          'service_worker'
```

But:
- Firefox does support event pages in manifest_version 3 extensions, and so the manifest should be valid
- Firefox does not yet support `background.service_worker` and it is only enabled when its separate about:config pref is set to true (tracked by #4605)

This issue is tracking fixing the addons-linter JSONSchema to make sure event pages used in `manifest_version: 3` extension manifests isn't reported as a linting error.

**Comment by ioanarusiczki** [comment]
@willdurand This seems to be working now, I tried the example above on 5.23.0 (I get the error on 5.22.0)

locally 

![locally](https://user-images.githubusercontent.com/33448286/203537116-5bd27dce-59c3-4150-92c1-6dfbf4b29a9f.png)

on AMO dev

![AMOdev](https://user-images.githubusercontent.com/33448286/203537151-d8f74a0b-aab2-4eca-8b6d-8c758d033e20.png)
